aries-site/src/examples/templates/forms/SignInExample.js

```diff
@@ -0,0 +1,171 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  Box,
+  Button,
+  CheckBox,
+  Form,
+  FormField,
+  Header,
+  Heading,
+  Layer,
+  Main,
+  Text,
+  TextInput,
+} from 'grommet';
+import { Close, Next } from 'grommet-icons';
+
+import { FormContainer } from '.';
+import { emailValidation } from './formHelpers';
+
+const ResetPassword = ({ closeLayer, email }) => {
+  const [formValues, setFormValues] = React.useState({ resetEmail: email });
+
+  // eslint-disable-next-line no-unused-vars
+  const onSubmit = ({ value, touched }) => {
+    // Your password reset logic here
+    // Display success status
+    closeLayer();
+  };
+
+  return (
+    <>
+      <Box
+        direction="row"
+        justify="end"
+        pad={{ horizontal: 'small', top: 'small' }}
+      >
+        <Button
+          a11yTitle="Close reset password form"
+          icon={<Close />}
+          onClick={closeLayer}
+        />
+      </Box>
+      <Box
+        gap="medium"
+        margin={{ horizontal: 'xlarge', bottom: 'xlarge', top: 'large' }}
+        width="medium"
+      >
+        <Heading level={2} margin="none">
+          Reset Password
+        </Heading>
+        <Form
+          validate="blur"
+          value={formValues}
+          onChange={setFormValues}
+          onSubmit={({ value, touched }) => onSubmit({ value, touched })}
+        >
+          <Text>A password reset email will be sent to </Text>
```

Comment:
Maybe we can look at another way to word this. I think having the label Email under this text is a little off and doesn't look right at least in my opinion.

Reply:
@britt6612 posted update. See if the following is working better for you:
![Screen Shot 2020-02-28 at 1 06 47 PM](https://user-images.githubusercontent.com/1756948/75583658-4a029c00-5a2b-11ea-81b5-d28567a58f7f.png)